packages/lodestar/src/network/gossip/validation/attestation.ts

```diff
@@ -120,20 +157,28 @@ export async function validateGossipAttestation(
 
 export async function isAttestingToInValidBlock(db: IBeaconDb, attestation: Attestation): Promise<boolean> {
   const blockRoot = attestation.data.beaconBlockRoot.valueOf() as Uint8Array;
-  //TODO: check if source and target blocks are not in bad block repository
+  // TODO: check if source and target blocks are not in bad block repository
   return await db.badBlock.has(blockRoot);
 }
 
-/**
- * is ready to be included in block
- */
-export function hasValidAttestationSlot(config: IBeaconConfig, genesisTime: number, attestation: Attestation): boolean {
-  const currentSlot = getCurrentSlot(config, genesisTime);
+export function isUnaggregatedAttestation(attestation: Attestation): boolean {
+  return Array.from(attestation.aggregationBits).filter((bit) => !!bit).length === 1;
+}
+
+export function isCommitteeIndexWithinRange(epochCtx: EpochContext, attestationData: AttestationData): boolean {
+  return attestationData.index < epochCtx.getCommitteeCountAtSlot(attestationData.target.epoch);
```

Comment:
method expects slot not epoch!!
Please create unit test for this:
```
@chainsafe/lodestar: 2020-10-06 21:34:14 [NETWORK]         error: Error while validating gossip attestation message=crosslink committee retrieval: out of range epoch: 1 
@chainsafe/lodestar: Error: crosslink committee retrieval: out of range epoch: 1
@chainsafe/lodestar:     at EpochContext._getSlotCommittees (/home/runner/work/lodestar/lodestar/packages/lodestar-beacon-state-transition/src/fast/util/epochContext.ts:236:13)
@chainsafe/lodestar:     at EpochContext.getCommitteeCountAtSlot (/home/runner/work/lodestar/lodestar/packages/lodestar-beacon-state-transition/src/fast/util/epochContext.ts:139:17)
@chainsafe/lodestar:     at isCommitteeIndexWithinRange (/home/runner/work/lodestar/lodestar/packages/lodestar/src/network/gossip/validation/attestation.ts:169:43)
@chainsafe/lodestar:     at Object.validateGossipAttestation (/home/runner/work/lodestar/lodestar/packages/lodestar/src/network/gossip/validation/attestation.ts:123:8)
@chainsafe/lodestar:     at GossipMessageValidator.isValidIncomingCommitteeAttestation (/home/runner/work/lodestar/lodestar/packages/lodestar/src/network/gossip/validator.ts:59:14)
@chainsafe/lodestar:     at LodestarGossipsub.validate (/home/runner/work/lodestar/lodestar/packages/lodestar/src/network/gossip/gossipsub.ts:91:26)
```

Reply:
ah yeah, good catch.  thanks for that.  will put this in next commit.